src/testFixtures/java/org/opensearch/neuralsearch/util/BatchIngestionUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.util;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A helper class to build docs for bulk request which is used by batch ingestion tests.
+ */
+public class BatchIngestionUtils {
+    private static final List<String> TEXTS = Arrays.asList(
+        "hello",
+        "world",
+        "an apple",
+        "find me",
+        "birdy",
+        "flying piggy",
+        "newspaper",
+        "dynamic programming",
+        "random text",
+        "finally"
+    );
+
+    public static List<Map<String, String>> prepareDocsForBulk(int startId, int count) {
```

Comment:
Please address @yuye-aws comments. 
nit: prepareDataForBulkIngestion

Reply:
I am OK with this function.